pyproject.toml

```diff
@@ -53,7 +53,7 @@ dependencies = [
     "pydot>=1.2.4",
     "pygtrie>=2.3.2",
     "pyparsing>=2.4.7",
-    "requests>=2.22",
+    "requests>=2.28",
```

Comment:
Unrelated change to fix studio tests

Reply:
Which one requires this update?

Reply:
This should be fixed in the `main` now (dependency update is not required). I suggest upgrading packages from time to time using:
```console
pip install --upgrade --upgrade-strategy eager -e ".[dev]"
```

This one is our dependency, but you won't run into mypy and other linting issue if you upgrade.

Reply:
Thanks @skshetry. Reverted. PTAL.